server/src/main/java/org/opensearch/cluster/coordination/ClusterBootstrapService.java

```diff
@@ -105,10 +114,10 @@ public ClusterBootstrapService(
         Consumer<VotingConfiguration> votingConfigurationConsumer
     ) {
         if (DiscoveryModule.isSingleNodeDiscovery(settings)) {
-            if (INITIAL_MASTER_NODES_SETTING.exists(settings)) {
+            if (INITIAL_CLUSTER_MANAGER_NODES_SETTING.exists(settings)) {
                 throw new IllegalArgumentException(
                     "setting ["
-                        + INITIAL_MASTER_NODES_SETTING.getKey()
+                        + INITIAL_CLUSTER_MANAGER_NODES_SETTING.getKey()
```

Comment:
Nitpick - assuming my prior comment on internal fallback is accurate, wouldn't messages like this be misleading as to what is driving the configuration/exception?

Reply:
Thanks for pointing out the error message. I addressed this along with the above issue. Now the setting name will be depends on which one (master or cluster_manager) has been set by user.